pkg/version/version.go

```diff
@@ -0,0 +1,22 @@
+/*
+   Copyright The Accelerated Container Image Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package version
+
+const (
+	OverlayBDVersionNumber = "0.1.0"
```

Comment:
Do we want to validate that this version corresponds to the installed version for the overlaybd tools?

Reply:
I don't think there is a correlation between overlaybd format version and the binary version...

Maybe adding release version support for specified overlaybd/fastoci format versions to README seems more reasonable ?

Reply:
That seems reasonable to me :)